core/rawdb/accessors_metadata.go

```diff
@@ -187,3 +187,16 @@ func WriteTransitionStatus(db ethdb.KeyValueWriter, data []byte) {
 		log.Crit("Failed to store the eth2 transition status", "err", err)
 	}
 }
+
+// WriteStateSizeMetrics writes the state size metrics to the database
+func WriteStateSizeMetrics(db ethdb.KeyValueWriter, data []byte) {
```

Comment:
I don't think we need to persistent the state size into database.
I would prefer to only keep them in memory.